daffodil-lib/src/test/scala/passera/test/UnsignedCheck.scala

```diff
@@ -30,14 +30,12 @@ import org.scalacheck._
 // import org.scalacheck.ConsoleReporter
 import org.scalacheck.Prop._
 import passera.unsigned._
-
 import org.junit.Test
 import org.junit.Assert._
 
 import scala.language.implicitConversions
 
 class UnsignedCheck {
-  import Gen._
```

Comment:
Removing an import should be safe as long as the code still compiles, but where is Gen defined?  I ran `rg -uu -w Gen` over my entire fully built checkout and it found only three lines, none of which appears to define Gen:

```
daffodil-lib/src/test/scala/passera/test/UnsignedCheck.scala
40:  import Gen._
52:  def genUInt: Gen[UInt] = for (n <- arbitrary[Int]) yield UInt(n)
61:  val nonNegLong = Gen.choose(0L, 0x00000000ffffffffL)
```

Reply:
IntelliJ removed this automatically I assume. 
Gen is defined in scalacheck/scalacheck_2.12/scalacheck_2.12-1.15.4-sources.jar!/org/scalacheck/Gen.scala